### ISSUE TITLE
Paste directly

### ISSUE THREAD

**Issue by friday** [friday/ulauncher-clipboard#4]
Since the reason for copying an entry is almost certainly to paste it, this could be made more efficient by pasting directly. Ulauncher doesn't support this, and from what I've seen, it doesn't seem very easy (and it's near impossible to search for this). You would have to know more about the GTK APIs than I currently do.

This could be the same keyword (replacing the behavior), a new keyword (keeping the old behavior), or a multiple-choice option, like [ulauncher-kill](https://github.com/Ulauncher/ulauncher-kill) (with alt+enter or on the main action)

Personally I would prefer both copying and pasting on the main action, and alt + enter could show a list of all actions (past, copy, delete).

**Comment by friday** [comment]
Closing, since it's [impossible](https://github.com/Ulauncher/Ulauncher/issues/311#issuecomment-451898815) unless you reply on a hack to send "ctrl+v", which doesn't work in Wayland (for good reasons) or for apps/users who uses something different :cry:

**Comment by friday** [comment]
Changing my mind already since this bugs me too much. I'll try to add a non-default option to send ctrl+v with `xdotools` after Ulauncher has closed (if the user has xdotools and x11)

**Comment by friday** [comment]
Fixed in 558994749681a166e99a4a412d3993ad4590d297

**Comment by yougg** [comment]
test OK with [dotool](https://git.sr.ht/~geb/dotool) on Gnome Wayland

hook command:

```bash
echo -e 'keydown rightctrl\nkey v\nkeyup rightctrl' | dotool
```
